frontend/class-opengraph.php

```diff
@@ -531,9 +531,15 @@ public function type( $echo = true ) {
 	/**
 	 * Create new WPSEO_OpenGraph_Image class and get the images to set the og:image.
 	 *
+	 * @param string|bool $image Optional. Image URL.
+	 *
 	 * @return void
 	 */
-	public function image() {
+	public function image( $image = false ) {
+		if ( empty( $image ) === false ) {
+			_deprecated_argument( __METHOD__, 'WPSEO 7.4' );
```

Comment:
https://github.com/Yoast/wpseo-video/blob/release/7.4/classes/class-wpseo-video-wrappers.php#L324
video is still using this in 7.4
Could you handle this please (create an issue, fix or whatever)?